pagerduty/resource_pagerduty_incident_workflow.go

```diff
@@ -430,7 +626,68 @@ func buildIncidentWorkflowInputsStruct(in interface{}) []*pagerduty.IncidentWork
 			Value: inputData["value"].(string),
 		}
 
+		generated := inputData["generated"].(bool)
+		if !generated {
+			specifiedInputNames = append(specifiedInputNames, input.Name)
+		}
 		newInputs[i] = &input
 	}
-	return newInputs
+	return newInputs, specifiedInputNames
+}
+
+func buildIncidentWorkflowInlineStepsInputsStruct(in interface{}) (
+	[]*pagerduty.IncidentWorkflowActionInlineStepsInput,
+	map[string][]*SpecifiedStep,
```

Comment:
These builder functions all now pass a pair of values as their result: the object that is eventually send as part of the request body, and the specified inputs tracking objects that can be referenced as we flatten the response.